servicex/servicex.py

```diff
@@ -465,6 +466,29 @@ async def get_data_rootfiles_uri_stream(
         ):  # type: ignore
             yield f_info
 
+    async def get_data_rootfiles_uri_async(
+        self,
+        selection_query: str,
+        title: Optional[str] = None,
+        as_signed_url: Optional[bool] = False,
+    ) -> list:
+        """Returns a list of each completed batch of work from ServiceX, taken from
+        the async iterator output of get_data_rootfiles_uri_stream.
+        The data that comes back includes a `url` that can be accessed to download the
+        data.
```

Comment:
The data that comes back is a list of `StreamInfoUrl`'s that contain a url for each output file from the transform - or something similar.